scipy/spatial/tests/test_kdtree.py

```diff
@@ -216,47 +216,22 @@ def test_vectorized_query_multiple_neighbors(self):
         assert_(np.all(~np.isfinite(d[:, :, -s:])))
         assert_(np.all(i[:, :, -s:] == self.kdtree.n))
 
-    @pytest.mark.parametrize('r', [0.8, 1.1])
-    def test_single_query_all_neighbors(self, r):
```

Comment:
Probably because my suggestion might have come across like that, sorry.

I agree we should keep the parametrization (if we keep testing r=0.9 & r=1.1)

Reply:
Okay, thanks - I have updated that.